x-pack/solutions/security/plugins/security_solution/common/api/endpoint/model/schema/common.schema.yaml

```
@@ -30,6 +32,43 @@ components:
     AgentId:
       type: string
       description: Agent ID
+    Kuery:
+      type: string
+      description: A KQL string.
+      example: "united.endpoint.host.os.name : 'Windows'"
+    HostStatuses:
+      type: array
+      description: A set of agent health statuses to filter by.
+      example: [ "healthy", "updating" ]
+      items:
+        type: string
+        enum:
+          - healthy
+          - offline
+          - updating
+          - inactive
+          - unenrolled
+    SortDirection:
+      type: string
+      description: Determines the sort order, which can be desc or asc.
```

```suggestion
      description: Determines the sort order.
```
We don't really need to mention this, since the enum field shows the possible values.